Dfe.Academies.External.Web/Models/ConversionApplicationContributor.cs

```diff
@@ -4,10 +4,11 @@ namespace Dfe.Academies.External.Web.Models;
 
 public class ConversionApplicationContributor
 {
-    public ConversionApplicationContributor(string firstName, string surname, SchoolRoles role, string? otherRoleName)
+    public ConversionApplicationContributor(string firstName, string surname, String emailAddress, SchoolRoles role, string? otherRoleName)
```

Comment:
shouldn't be lower case string otherwise c# thinks you're using the type definition?